ert3/storage/_storage.py

```diff
@@ -399,11 +430,14 @@ def _get_experiment_parameters(
         raise ert3.exceptions.StorageError(response.text)
     parameters: MutableMapping[str, List[str]] = {}
     for name in response.json():
-        key, val = name.split(".")
-        if key in parameters:
-            parameters[key].append(val)
+        if len(name.split(".")) == 1:
```

Comment:
I assume this is to separate between `blob` and `numerical` records? If so, I suggest we do so via a boolean flag or function that hints towards this.